docs/notebooks/units.ipynb

```diff
@@ -78,7 +79,69 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Threshold indices\n",
+    "## Smart conversions : Precipitation\n",
+    "\n",
+    "For precipitation data, `xclim` usually expects precipitation fluxes, so units of `mass / (area * time)`. However, many indicators will also accept rates (`length / time`, for example `mm/d`) by implicitly assuming the data refers to liquid water, and thus that we can simply multiply by 1000 kg/m³ to convert from the latter to the former. This transformation is enabled on indicators that have `Indicator.context == 'hydro'`.\n",
+    "\n",
+    "We can also leverage the CF conventions to perform some other \"smart\" conversions. For example, if the CF standard name of an input refers to liquid water, the flux <-> rate and amount <-> thickness conversions explained above will be automatic in `xc.core.units.convert_units_to`, whether or not the \"hydro\" context is activated. Another CF-driven conversion is between amount and flux or thickness and reate. Here again, `convert_units_to` will see if the  `standard_name` attribute allors and will need to infer the frequency of the data. For example, if a daily precipitation series records total daily precipitation and has units of `mm`, it should use the `lwe_thickness_of_precipitation_amount` standard name and have a regular time coordinate, so that we can understand and convert it to a precipitation flux. These conversions are however not automatically done in the indicator and  `convert_units_to` should be called beforehand. \n",
```

Comment:
reate -> rate
allors ?

The beginning of the paragraph is clear, but the end gets confusing.

Reply:
Woups indeed, I committed a partial edit. It is updated.